specs/Standards/NonFungibleToken/Payout.md

```diff
@@ -66,35 +72,43 @@ Financial contracts MAY take a cut of the NFT sale price as commission, subtract
 #[derive(Serialize, Deserialize)]
 #[serde(crate = "near_sdk::serde")]
 pub struct Payout {
-  pub payout: HashMap<AccountId, U128>,
+  payout: HashMap<AccountId, U128>,
 }
 
-pub trait Payouts{
+pub trait Payouts {
   /// Given a `token_id` and NEAR-denominated balance, return the `Payout`.
   /// struct for the given token. Panic if the length of the payout exceeds
   /// `max_len_payout.`
-  fn nft_payout(&self, token_id: String, balance: U128, max_len_payout: u32) -> Payout;
+  fn nft_payout(&self, token_id: String, balance: U128, max_len_payout: Option<u32>) -> Payout;
   /// Given a `token_id` and NEAR-denominated balance, transfer the token
   /// and return the `Payout` struct for the given token. Panic if the
   /// length of the payout exceeds `max_len_payout.`
-  #[payable]
   fn nft_transfer_payout(
     &mut self,
     receiver_id: AccountId,
     token_id: String,
-    approval_id: u64,
+    approval_id: Option<u64>,
+    memo: Option<String>,
     balance: U128,
-    max_len_payout: u32,
-  ) -> Payout{
+    max_len_payout: Option<u32>,
+  ) -> Payout {
     assert_one_yocto();
```

Comment:
This can be dropped as it does exactly the same as the `payable` attribute.

Reply:
```suggestion
```